integration-tests/integration/receptionMove/receptionConfirmMovePage.cy.js

```diff
@@ -0,0 +1,178 @@
+const receptionConfirmMovePage = require('../../pages/receptionMove/receptionConfirmMovePage')
+const offenderFullDetails = require('../../mockApis/responses/offenderFullDetails.json')
+const offenderBasicDetails = require('../../mockApis/responses/offenderBasicDetails.json')
+
+const offenderNo = 'G3878UK'
+
+before(() => {
+  cy.clearCookies()
+  cy.task('resetAndStubTokenVerification')
+  cy.task('stubSignIn', {
+    username: 'ITAG_USER',
+    caseload: 'MDI',
+    roles: ['ROLE_CELL_MOVE'],
+  })
+  cy.signIn()
+  cy.task('stubOffenderFullDetails', {
+    ...offenderFullDetails,
+    alerts: [
+      {
+        alertId: 6,
+        alertType: 'X',
+        alertTypeDescription: 'Security',
+        alertCode: 'XGANG',
+        alertCodeDescription: 'Gang Member',
+        dateCreated: '2023-10-10',
+        expired: false,
+        active: true,
+        addedByFirstName: 'GURNANK',
```

Comment:
I'f prefer for my name not to be in tests!

Reply:
Good spot, thanks